bindings/python/crocoddyl/core/diff-action-base.hpp

```diff
@@ -57,10 +57,22 @@ class DifferentialActionModelAbstract_wrap : public DifferentialActionModelAbstr
 
   boost::shared_ptr<DifferentialActionDataAbstract> default_createData() {
     return this->DifferentialActionModelAbstract::createData();
-  }
-};
+    void quasiStatic(const boost::shared_ptr<DifferentialActionDataAbstract>& data, Eigen::Ref<Eigen::VectorXd> u,
+                     const Eigen::Ref<const Eigen::VectorXd>& x, const std::size_t& maxiter, const double& tol) {
```

Comment:
You should also define the default argument of `maxiter`, `tol` as done in the c++ interface. This will require to adapt the `.def` function in the `.cpp`